app/javascript/src/components/Dashboard/Notes/Pane/Edit.jsx

```diff
@@ -7,14 +7,26 @@ import Form from "./Form";
 const EditNotePane = ({ fetchNotes, showPane, setShowPane, note }) => {
   const onClose = () => setShowPane(false);
 
+  const convertNoteToFormFormat = note => ({
+    title: note.title,
+    description: note.description,
+    contact: null,
+    tags: [],
+  });
```

Comment:
Move this to an utils.js within the closest scope. We should try our level best to keep the presentation logic as clean as possible.

Reply:
Will fix it ASAP.

Reply:
Fixed it.

Reply:
Moved convertNoteToFormFormat to utils.jsx in Contacts folder and changed all pertinent imports.